### DISCUSSION TITLE
Design ideas / ideas dump

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system#188] [Ideas]
For quick notes on discussions around things we could include into our design system to improve

**Comment by michaelcattell** [comment]
I've noticed on each pattern from a handful of differing design systems with a "last updated: date" text underneath the title of the pattern, where as they then link (a couple of them) to the discussion/git issue which feels a good move
(Reason: i think it'll help in the long run when collecting newer updates to the pattern and getting traction/self-serving within discussions)

Examples: 
**MOJ**
![image](https://user-images.githubusercontent.com/52162960/129706448-47561e56-2465-4fd1-a353-0e688d36d36e.png)

**CJS Crime Programme**
![image](https://user-images.githubusercontent.com/52162960/129706609-b61b38fa-e064-40f3-951a-2777cc280d51.png)

**Comment by jonhurrell** [comment]
Can we use the design community to shape the work of an up coming component/pattern from the backlog, ready for us to bring it into the design system.
e.g: a mini sprint of designers (10% time?) to answer

- Does the user need hold up?
- What exists in different service and why?
- What don't we know?
- Any new or existing evidence to support a preference?
- Any insights from the group to feed into the design system documentation?

**Reply by michaelcattell** [reply]
Related to above.

Feedback from MOJ (Michael Chambers - Designer in the DesignOps team), they gather information by the community by asking their contributors to fill in a google doc with these questions:

Sub title at the top: Fill out the sections you feel comfortable, if not get in touch with the enablers (enablers are people from the design community who volunteer their time to help assist the design library)

**What is Existing already?**
(Link similar)

**What research has been done?**

**When to use it**

**How to use it**

**Things we don’t know enough?**
(what’s not perfect)

Once done - they then get content designer involved to rewrite for pattern page and upload to the design system (put live) and update the communities that this pattern is in-need of help and support and ask for feedback/research etc.

If a pattern has been sitting with no contributes from the above initial request it is brought up in their fortnightly-run Power Hour sessions and discussed as a driver.

**Comment by michaelcattell** [comment]
Work with @HelenOsg on amending this section of the patterns/components to help users direct feedback easily:
![image](https://user-images.githubusercontent.com/52162960/130807537-b747a5aa-d4c2-49e6-bfac-9cf8742de655.png)
(things like links to the discussions/github issue etc)
